service/frontend/admin_handler.go

```diff
@@ -1150,7 +1150,7 @@ func (adh *AdminHandler) AddOrUpdateRemoteCluster(
 			HistoryShardCount:        resp.GetHistoryShardCount(),
 			ClusterId:                resp.GetClusterId(),
 			ClusterAddress:           request.GetFrontendAddress(),
-			HttpAddress:              request.GetFrontendHttpAddress(),
```

Comment:
Why are we taking the frontend grpc address from the request an the http address from the `resp`?

Reply:
we need to know the grpc address to connect two clusters. From my understanding, we don't have http endpoint on admin service

Reply:
I think we should be consistent here.

Reply:
We synced offline, this makes sense to me now.

You could argue that the frontend address (gRPC) should be taken from the response too but it's okay the way it is.